package.json

```diff
@@ -62,7 +55,8 @@
     "tape": "^5.3.2",
     "tsify": "^5.0.4",
     "typedoc": "^0.22.10",
-    "typescript": "^4.5.2"
+    "typescript": "^4.5.2",
+    "util": "^0.12.4"
```

Comment:
This isn't necessary since we run tests only in Node environment which does have `util`. If TS complained it was because `tsconfig.test.json` was missing Node env setup from `lib`.